proposals/NNNN-replace-pthread-implementation-with-threads-from-cpp11.md

```diff
@@ -0,0 +1,80 @@
+# Replace thread::Thread implementation with std::thread from C++11
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [SDL Developer](https://github.com/smartdevicelink)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+Replacing [pthreads](https://man7.org/linux/man-pages/man7/pthreads.7.html) with [std::thread](https://www.cplusplus.com/reference/thread/thread/) will make the code more platform-independent. That will simplify build, support and porting of the code to various platforms.
+
+## Motivation
+
+Currently we are using pthread library (in POSIX systems) for working with threads. It is a C library which does not consider some OOP aspects critical to C++ like an object lifetimes and exceptions handling.
```

Comment:
The current implementation has...
>>It is a C library which does not consider some OOP aspects critical to C++ like an object lifetimes and exceptions handling.

re-write the advantages of the chosen approach